qt/scientific_interfaces/ISISReflectometry/GUI/RunsTable/RunsTablePresenter.cpp

```diff
@@ -310,6 +325,13 @@ void RunsTablePresenter::ensureAtLeastOneGroupExists() {
   // group first, then delete the original one
   appendEmptyGroupInView();
```

Comment:
Are these two lines still required? The original fix here was to remove a "bad" group and replace it with a proper one, but this is being removed below now anyway?

Reply:
It doesn't work without it due to JobTreeView always having at least 1 row in it, and then it failing at attempting to erase it in the following line. Currently, it has to repair to then delete if that makes sense?